doc_admin/AdminManual.adoc

```diff
@@ -39,83 +48,77 @@ This application software consists of the following parts:
 
-The real part of the installation includes installing the JRE (Java Runtime Environment), installing the MySQL Server, and the creation of the database (DB). This installation is accomplished through the execution of a script in the folder sql/ within the .zip file folders.
+The procedure includes the installation of the JRE (Java Runtime Environment), the MariaDB/MySQL Server and the creation of the database through the execution of a script in the folder sql/ within the .zip file folders.
 
-The classic installation schema in a regular wired LAN network is shown in the following figure:
+The typical Client / Server, networked (LAN) configuration is shown in the following figure:
 
 image:image3.png[image,width=642,height=243]
 
 The Administrator/Installer tasks are hereby listed:
 
 TIP: If when running Open Hospital the application appears small-scaled or blurry then the issue is probably using a high-dpi display. In such cases there are some solutions (https://stackoverflow.com/a/39372897/5309541[click here]) to apply only on the machines using this type of monitor.
 
-* Install the Java Runtime Environment 1.8 or higher (1.8 is recommended for full compliance)
-* Install the MySQL server on a computer that will act as the OH “server”
+* Install the Java Runtime Environment (see compatibility matrix for the right version)
+* Install the MariaDB/MySQL server on a computer that will act as the OH server
 * Copy the Open Hospital application on every client (it may be installed on the server also)
-* Configure each Open Hospital copy to point to the MySQL server (using the hostname or the IP address)
+* Configure each Open Hospital copy to point to the OH server (using the hostname or the IP address)
 
 For the Java Runtime Environment installation refer to the next chapter <<jre,Java Runtime Environment>>
 and the online documentation.
 
-For the MySQL installation and DB creation refer to the chapter <<mysql-server,MySQL Server and Open Hospital DB>> and the online documentation.
+For the MariaDB/MySQL database server installation and database creation refer to the chapter <<database-server,Database Server and Open Hospital DB>> and the online documentation.
 
 For the Open Hospital installation on every client, it is sufficient to copy and extract the zip file on the machine.
 
 For the Open Hospital configuration refer to the chapter <<config-files,Configuration Files>>.
 
+TIP: Some tasks can be automated using the new oh.sh / oh.bat scripts.
+
 ==== 3.1.1 Prerequisites - Software compatibility matrix
 
 Before starting, check the following software compatibility matrix, in order to verify the external software versions needed to run Open Hospital and the current available extended features for every version, architecture and operating system.
```

Comment:
"in order" can be removed as it is overly wordy and not needed

Reply:
yes :-)